fastlane/metadata/en-CA/subtitle.txt

```diff
@@ -1 +0,0 @@
-Design a site, build a blog
```

Comment:
This and all the `en-CA` files above were deleted because of a 429 from GlotPress:

```
[13:48:03]: No translation available for en-CA
```

I'm curious why it seems to be always `en-CA` that gets it. Same for the `.strings`, I've got it on `da` quite a few times.

Reply:
🤔 

One thing they have in common is that they are both the sixth file requested:

```
[13:46:18]: Downloading translations for 'ar' from GlotPress (ar) [{:status=>"current"}]...
[13:46:20]: Downloading translations for 'bg' from GlotPress (bg) [{:status=>"current"}]...
[13:46:22]: Downloading translations for 'cs' from GlotPress (cs) [{:status=>"current"}]...
[13:46:24]: Downloading translations for 'cy' from GlotPress (cy) [{:status=>"current"}]...
[13:46:26]: Downloading translations for 'da' from GlotPress (da) [{:status=>"current"}]...
[13:46:27]: Error downloading locale `da` — 429 Too Many Requests (https://translate.wordpress.org/projects/apps/ios/dev/da/default/export-translations/?filters%5Bstatus%5D=current&format=strings)
```

```
Downloading language: ar-SA
Downloading language: da
Downloading language: de-DE
Downloading language: en-AU
Downloading language: en-CA
[13:48:03]: No translation available for en-CA
```